src/models/media/container.yaml

```
@@ -0,0 +1,8 @@
+type: object
+required:
+  - container
+properties:
+  container:
+    type: string
+    description: "Container format of the media."
+    example: "mp4" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)